src/01-dinosaur-facts.js

```diff
@@ -22,7 +22,18 @@ const exampleDinosaurData = require("../data/dinosaurs");
  *  getLongestDinosaur(dinosaurs);
  *  //> { Brachiosaurus: 98.43 }
  */
-function getLongestDinosaur(dinosaurs) {}
+const getLongestDinosaur = (dinosaurs) => {
+  if (dinosaurs.length === 0) {
+    return {};
+  }
+  const longestDinosaur = dinosaurs.reduce((longest, current) => {
+    return current.lengthInMeters > longest.lengthInMeters ? current : longest;
+  })
+  const {name, lengthInMeters} = longestDinosaur;
+  const lengthInFeet = lengthInMeters * 3.281;
```

Comment:
it could be easier to read if we did all this extra work here in a helper funciton

imagine after reduce we simply `return _getLengthInFeet(longestDinosaur)`